doc/active_docs/Account Management API.json

```diff
@@ -9935,5 +9929,14 @@
       }
     }
   },
-  "components": {}
+  "components": {
+    "schemas":{
+      "Permissions":{
+        "type": "array",
+        "items": {
+          "$ref": "#/components/schemas/Permissions"
```

Comment:
I don't get it... The items reference the same schema, but what are the elements?... 

Swagger UI apparently interprets it as an array of strings, but this definition does not look right.

Reply:
you mean defining properties with type and description